tests/schema/tests.py

```diff
@@ -454,9 +454,9 @@ def test_alter_text_field_to_time_field(self):
         """
         with connection.schema_editor() as editor:
             editor.create_model(Note)
-        Note.objects.create(info='3:16')
+        Note.objects.create(info='3:16:17.4567')
         old_field = Note._meta.get_field('info')
-        new_field = TimeField(blank=True)
+        new_field = TimeField(blank=True, null=True)
```

Comment:
Given the failure I've seen, I'd like to see a similar test where `new_field` has `null=False` and a default.